tests/tt_metal/multihost/multi_host_tests/CMakeLists.txt

```
@@ -0,0 +1,24 @@
+# Smoke tests (fast, not necessarily thorough)
+add_library(mullti_host OBJECT)
```

Seems like you have some spelling errors, here, boss
```suggestion
add_library(multi_host OBJECT)
```

---

ouch! thanks!